contrail-overlay.nix

```diff
@@ -119,6 +119,7 @@ let
             "path"
             "deps"
             "dev"
+            "build-webui-deps"
```

Comment:
Doesn't work because the target is `webui.build-webui-deps` and this code only exclude top level attrs.

Reply:
That one has been moved and is now part of lib and thus ignored through the sanitizing